auth/src/test/java/com/firebase/ui/auth/compose/configuration/AuthUIConfigurationTest.kt

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2025 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.firebase.ui.auth.compose.configuration
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccountCircle
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.auth.actionCodeSettings
+import org.junit.Assert.assertThrows
+import org.junit.Test
+import java.util.Locale
+import kotlin.reflect.KMutableProperty
+import kotlin.reflect.full.memberProperties
+
+class AuthUIConfigurationTest {
```

Comment:
Would like to see a test for the Builder pattern as well

Reply:
not sure if i understand, doesn't the authUIConfiguration dsl function tests handle the builder pattern test also?